### ISSUE TITLE
AcceptWorldGuard

### ISSUE THREAD

**Issue by MithrandirCraft** [BerndiVader/mmCustomSkills26#10]
The feature is undocumented. But you told me this addon has it right?
On my server, when I spawn a MythicMob inside a worldguard region with the deny-spawn flag set to true, the mob doesn't spawn.
In fact, some mobs can't even spawn even if they are inside a worldguard region which allows mobspawning.
Could I use this feature in order to force the bypassing of worldguard regions for those mobs using:
  "AcceptWorldGuard: false"?
Because apparantly this isn't working that way, and I'm now realizing it may have been ment for the opposite effect for some reason (Guaranteeing that the mobs don't spawn inside denied wg regions).

In conclusion: I'd like to ALLOW mobs to spawn in such regions setting the feature to false. But it isn't working.

**Comment by muhahahahe** [comment]
sry Mith it is obfuscated

but u can also use the wgdenyspawnflag or stateflag conditions in ur spawners


*** closed ***

**Comment by muhahahahe** [comment]
sry Mith its obsolete

blame xikage for the spawners :P

also u can try using wgdenyspawn or wgstateflag as a condition maybe u can figure something out
